firefox-ios/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -138,7 +138,7 @@
 "Join 15 million people growing the right trees in the right places." = "Join 20 million people growing the right trees in the right places.";
 "sustainable shoes" = "sustainable shoes";
 "Trees planted" = "Trees planted";
-"Before ..." = "Before ...";
+"Before ..." = "Before...";
```

Comment:
Does this string need a double-check? Noticing the difference I wonder whether we'd need a new matching "key" as well